tests/ga/test_monitor.py

```diff
@@ -98,16 +105,21 @@ def test_it_should_invoke_all_periodic_operations(self):
             def mock_run(self):
                 invoked_operations.append(self._name)
 
-            with patch.object(PeriodicOperation, "run", side_effect=mock_run, spec=MonitorHandler.run):
+            with patch.object(PeriodicOperation, "run", side_effect=mock_run, autospec=True):
```

Comment:
the spec was coming from MonitorHandler.run instead of PeriodicOperation.run; using autospec instead